docs/source/admin/upgrading.rst

```diff
@@ -3,30 +3,29 @@ Upgrading JupyterHub
 ====================
 
 JupyterHub offers easy upgrade pathways between minor versions. This
-document describes how to do these upgrades.
+document describes how to make these upgrades.
 
 If you are using :ref:`a JupyterHub distribution <index/distributions>`, you
 should consult the distribution's documentation on how to upgrade. This
 document is useful if you have set up your own JupyterHub without using a
 distribution.
 
-The steps are discussed in detail, so if you get stuck at any step you can always refer to this guide. Most likely, 
-upgrading JupyterHub is painless, quick and with minimal user interruption.
+The steps are discussed in detail, so if you get stuck at any step you can always refer to this guide. Upgrading JupyterHub is usually painless, quick and with minimal user interruption.
 
 Read the Changelog
 ==================
 
 The `changelog <../changelog.md>`_ contains information on what has
-changed with the new JupyterHub release and any deprecation warnings.
-Read these notes to familiarize yourself with the upcoming changes. There
```

Comment:
I think "with" perfectly fits in as they are mostly talking about the changes happened with the release of new JupyterHub